pages/api/inngest.ts

```diff
@@ -106,22 +106,36 @@ const periodicZap = inngest.createFunction(
         },
       });
 
-      const { nostrWalletConnectUrl, recipientLightningAddress, amount } =
-        subscription;
+      const {
+        nostrWalletConnectUrl,
+        recipientLightningAddress,
+        amount,
+        currency,
+      } = subscription;
       const message = subscription.message ?? undefined;
 
       let paymentSucceeded = false;
       let paymentRecovered = false;
       let errorMessage = "";
       try {
+        let satoshi = amount;
+        if (currency && currency !== SATS_CURRENCY) {
+          satoshi = await fiat.getSatoshiValue({
+            amount,
+            currency,
+          });
+          logger.info("calculated satoshi amount for fiat payment", {
+            subscriptionId,
+            currency,
```

Comment:
`currency` is excessive, or `amount` would also be useful.

Reply:
I will add amount too, thanks